src/main/java/reposense/util/FileUtil.java

```diff
@@ -57,44 +63,55 @@ public static void deleteDirectory(String root) throws IOException {
         }
     }
 
-    public static void unzip(ZipInputStream zipInput, String destinationFolder) {
-        Path directory = Paths.get(destinationFolder);
-
-        // buffer for read and write data to file
-        byte[] buffer = new byte[2048];
-
-        try {
-            Files.createDirectories(directory);
-            ZipEntry entry = zipInput.getNextEntry();
+    /**
+     * Zips all the JSON files contained in the {@code sourcePath} directory.
+     * Creates the zip folder in the {@code outputPath}
+     */
+    public static void zipJson(String outputPath, String sourcePath) {
+        List<Path> allJsonFiles = getFilePath(outputPath + File.separator + sourcePath, ".json");
```

Comment:
Put this inside the `try` block as well, then `getFilePaths(..)` can just throw the `IOException` up here.